UX-Guide-Metadata/draft/techniques/onix-metadata/index.html

```diff
@@ -1062,9 +1076,8 @@ <h5>Instructions</h5>
 							<ol class="condition">
 								<li><b>LET</b> <var>adaptation_string</var> be the result of:
 									<ul class="condition">
-										<li>calling join on <var>adaptation</var> with the separator <code>", "</code></li>
-										<li>making uppercase the first character</li>
-										<li>replacing the last occurrence of <code>", "</code> with <code id="join-array-and">", and "</code>.</li>
+										<li>calling <a href="#join-array-to-comma-list">join array to comma list</a> given <var>adaptation</var></li>
+										<li>making uppercase the first character.</li>
```

Comment:
Where does that uppercase of the first character happen?
I don't see that in the function itself nor its example of
`<p>For example if the input array has elements <code>["one", "two", "three"]</code>, then the output is going to be <code>"one, two, and three"</code>.</p>`

Reply:
This seems okay to me. This step uppercases the string returned from the function.

You might just want to say "uppercasing the first character" for simplicity, though.

Reply:
Done, thank you